prow/config.yaml

```diff
@@ -1104,6 +1104,70 @@ presubmits:
           path: /mnt/disks/ssd0/docker-graph
       - name: var-lib-docker
         emptyDir: {}
+  - name: pull-kubernetes-e2e-containerd-gce
+    agent: kubernetes
+    context: pull-kubernetes-e2e-containerd-gce
+    rerun_command: "/test pull-kubernetes-e2e-containerd-gce"
+    trigger: "(?m)^/test pull-kubernetes-e2e-containerd-gce,?(\\s+|$)"
+    always_run: false
+    skip_report: false
+    branches:
+    - master
+    spec:
+      containers:
+      - args:
+        - --root=/go/src
+        - --repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)
+        - --repo=k8s.io/release
+        - --repo=github.com/kubernetes-incubator/cri-containerd
+        - --upload=gs://kubernetes-jenkins/pr-logs
+        - --git-cache=/root/.cache/git
+        - --clean
+        - --timeout=90
+        - --
+        - --build=bazel
+        - --mode=local
+        - --runtime-config=batch/v2alpha1=true,admissionregistration.k8s.io/v1alpha1=true
```

Comment:
this line and the two above should be moved to config.json

Reply:
and then you can also remove `- --`

Reply:
Done